app/_locales/en/messages.json

```diff
@@ -1402,6 +1402,12 @@
   "importMyWallet": {
     "message": "Import My Wallet"
   },
+  "importNFTAddressToolTip": {
+    "message": "On OpenSea, for example, on the NFT's page under Details, there is a blue hyperlinked value labeled 'Contract Address'. If you click on this, it will take you to the contract's address on Etherscan; at the top-left of that page, there should be an icon labeled 'Contract', and to the right, a long string of letters and numbers. This is the address of the contract that created your NFT. Click on the 'copy' icon to the right of the address, and you'll have it on your clipboard."
+  },
+  "importNFTTokenIdToolTip": {
+    "message": "A collectible's ID is a unique identifier since no two NFTs are alike. Again, on OpenSea this number is under 'Details'. Make a note of it, or copy it onto your clipboard."
+  },
```

Comment:
Has this copy been provided by design? I'm not sure of the process on how we write/edit copy for the app I'm assuming in a lot of cases we just come up with it? Seems like it could be improved but I don't have any suggestions right now. Also seems like a lot of copy for a tooltip. I know, not the most helpful comment.

<img width="457" alt="Screen Shot 2022-01-19 at 8 03 26 PM" src="https://user-images.githubusercontent.com/8112138/150080951-74054e8b-1ee0-4b86-9d14-5068e7ce734e.png">
<img width="466" alt="Screen Shot 2022-01-19 at 8 03 56 PM" src="https://user-images.githubusercontent.com/8112138/150080958-cc62b34c-7293-4b69-b317-21c94cfae939.png">